widgetnode.py

```
@@ -0,0 +1,92 @@
+from __future__ import annotations
+from dataclasses import asdict, dataclass, field
```

Ruff (0.8.2)

`dataclasses.asdict` imported but unused

Remove unused import: `dataclasses.asdict`

(F401)